Firestore/core/src/firebase/firestore/local/persistence.h

```diff
@@ -17,9 +17,13 @@
 #ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_LOCAL_PERSISTENCE_H_
 #define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_LOCAL_PERSISTENCE_H_
 
+#include <functional>
+#include <memory>
```

Comment:
This does not appear to use anything from `<memory>`. Remove?

Reply:
Done.